mlrun/api/crud/model_monitoring/helpers.py

```diff
@@ -127,3 +127,27 @@ def get_stream_path(project: str = None):
     return mlrun.common.model_monitoring.helpers.parse_monitoring_stream_path(
         stream_uri=stream_uri, project=project
     )
+
+
+def get_connection_string(project: str = None) -> str:
```

Comment:
Also, `get_connection_string` is a bad name for this function, since it doesn't return a connection string - it returns a provider. Should be something like `get_project_secret_provider`.

Reply:
fixed
In the future this function can be a general secret provider function for the API side. Right now it only useful for the endpoint connection string but we can easily convert it to a genre secret provider function.